pkg/net/url_test.go

```diff
@@ -107,12 +107,13 @@ func TestURLUnmarshalJSON(t *testing.T) {
 		{[]byte(`"https://play.min.io:0"`), &URL{Scheme: "https", Host: "play.min.io:0"}, false},
 		{[]byte(`"https://147.75.201.93:9000/"`), &URL{Scheme: "https", Host: "147.75.201.93:9000", Path: "/"}, false},
 		{[]byte(`"https://s3.amazonaws.com/?location"`), &URL{Scheme: "https", Host: "s3.amazonaws.com", Path: "/", RawQuery: "location"}, false},
-		{[]byte(`"http://myminio:10000/mybucket//myobject/"`), &URL{Scheme: "http", Host: "myminio:10000", Path: "/mybucket/myobject"}, false},
+		{[]byte(`"http://myminio:10000/mybucket//myobject"`), &URL{Scheme: "http", Host: "myminio:10000", Path: "/mybucket/myobject"}, false},
```

Comment:
This case is already covered in below test case. you could need to add new test case for ends with `//`

Reply:
Done :+1: